src/components/v5/common/ActionSidebar/partials/DecisionMethodRow/DecisionMethodRow.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { type FC } from 'react';
+
+import { formatText } from '~utils/intl.ts';
+import ActionFormRow from '~v5/common/ActionFormRow/index.ts';
+import { FormCardSelect } from '~v5/common/Fields/CardSelect/index.ts';
+
+import { useDecisionMethods } from '../../hooks/index.ts';
+
+import { type DecisionMethodRowProps } from './types.ts';
+
+const DecisionMethodRow: FC<DecisionMethodRowProps> = ({
```

Comment:
Please remove the `Row` from the variables naming (both component and prop) as it's redundant

Reply:
@rdig I wanted to make it consistent with other fields:
![image](https://github.com/JoinColony/colonyCDapp/assets/44369011/c18df7cd-1169-45da-bca3-4da0fda4ef54)

Should I remove `Row` from `DescriptionRow` and `CreatedInRow` components as well or can I leave the `Row` everywhere?

Reply:
I'd personally like to have 'row' removed from all those components.

So please rename `CreateInRow` as well.

Thanks.